apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -91,7 +90,7 @@ class AnimationPickerListItem extends React.Component {
       }
     ];
     const iconImageSrc = this.props.category
-      ? this.props.categoryImagePathPrefix + `${this.props.category}.png`
+      ? `/blockly/media/animation-previews/category_${this.props.category}.png`
```

Comment:
i think we still need the "lab" directory in this path -- `/blockly/media/p5lab/animation-previews`

Reply:
aw shoot I put it in the wrong place. I'll fix that